pom.xml

```diff
@@ -146,8 +146,7 @@
         <aspectjweaver.version>1.7.4</aspectjweaver.version>
         <casclient.version>3.2.2</casclient.version>
         <ccpp.version>1.0</ccpp.version>
-        <cernunnos.version>1.2.2</cernunnos.version>
-        <cglib.version>3.1</cglib.version>
+        <cernunnos.version>1.3.0-SNAPSHOT</cernunnos.version>
```

Comment:
Gary R is working with Drew to get an updated version of cernunnos published that will allow us to resolve this.